content/agent/_index.md

```diff
@@ -3,103 +3,132 @@ title: Agent
 kind: documentation
 description: Install & configure the Agent to collect data
 aliases:
-    - /guides/basic_agent_usage/
+    - /agent/faq/agent-status-and-information
+    - /agent/faq/start-stop-restart-the-datadog-agent
+    - /agent/faq/send-logs-and-configs-to-datadog-via-flare-command
+    - /agent/faq/how-to-get-more-logging-from-the-agent
 ---
 
-{{< partial name="platforms/platforms.html" >}}
+<div class="alert alert-info">
+    Agent v6 is now available, <a href="https://github.com/DataDog/datadog-agent/blob/master/docs/agent/upgrade.md">upgrade to the newest version </a> to benefit from all new functionalities 
```

Comment:
s/to benefit from all new functionalities/to benefit from all new functionality./

Reply:
👌